certified-connectors/GroupMgr/apiDefinition.swagger.json

```diff
@@ -10,7 +10,7 @@
       "email": "support@groupmgr.com"
     }
   },
-  "host": "vragroupmanagerapprovalflow.azurewebsites.net",
+  "host": "vragroupmanagerapprovalflow-h8tvt-uzi02fbsspubp3eg.azurewebsites.net",
```

Comment:
@Jwaegebaert Please let us if it is a test or stage host ? If yes, we do not accept test/stage/preview host URLs for certification.

Reply:
@sriyen-msft This is our official production host URL. The previous one was our test host.